guides/v9-hammerjs-migration.md

```diff
@@ -129,14 +129,16 @@ plugin, or to an actual `@Output`. For example:
 
 In the example above, `rotate` could be an event from the deprecated `GestureConfig`, or an
 `@Output` from `<image-rotator>`. The migration warns about this to raise awareness that it
-might have _incorrectly kept_ HammerJS. Please manually check if you can remove HammerJS.
+might have _incorrectly kept_ HammerJS. Please check if HammerJS can be removed from the project
+manually.
```

Comment:
My intention was to use active voice since we talked about the problems of passive voice in a tech-talk. I'm not sure though if I got it right 😄

Reply:
I think that you did a good job with active vs passive voice.

However, at the start of the doc there was very little you/your, but towards the end there was a lot. I just wanted to make it consistent. Should I make it consistently use you/your instead?